client/js/components/day/employee.js

```diff
@@ -17,10 +17,16 @@ var Employee = React.createClass({
 
   // connectDragSource added to props by DragSource
   render: function(){
+    var ratings = ["#F26350", "#F26350", "#6BCFFF", "#6BCFFF", "#6BCFFF", "#A8E5A7"];
```

Comment:
What do you mean exactly? In Kyle's example data object, the ratings are values 0-5 so if I were to map the colors to an object the keys would just be those values. Happy to adjust just want to make sure I know what you mean.